package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@tscircuit/cli",
   "main": "dist/main.js",
   "type": "module",
-  "version": "0.1.45",
+  "version": "0.1.47",
```

Comment:
You don't have to change the version, remove this

Reply:
Done @imrishabh18

Reply:
Done @imrishabh18